test/functional/feature_fee_estimation.py

```diff
@@ -243,5 +244,11 @@ def run_test(self):
         self.log.info("Final estimates after emptying mempools")
         check_estimates(self.nodes[1], self.fees_per_kb)
 
+        self.log.info("Testing that fee estimation is disabled in blocksonly.")
```

Comment:
18766 too?

Reply:
Resolved backport PR commit fusion in latest push